lib/net/http/persistent.rb

```diff
@@ -447,6 +449,21 @@ def self.detect_idle_timeout uri, max = 10
 
   attr_reader :verify_mode
 
+  ##
+  # HTTPS verify_hostname.
+  #
+  # If a client sets this to true and enables SNI with SSLSocket#hostname=,
+  # the hostname verification on the server certificate is performed
+  # automatically during the handshake using
+  # OpenSSL::SSL.verify_certificate_identity().
+  #
+  # You can set +verify_hostname+ as true to use hostname verification
+  # during the handshake.
+  #
+  # NOTE: This may work with Ruby > 2.8.
```

Comment:
Ruby 2.8 is not a released version, perhaps this can be rephrased?

Reply:
Yep. Ruby 3.0 or later should be correct.
I'll take a glance later.

Reply:
Fixed in https://github.com/drbrain/net-http-persistent/pull/120/commits/123f97da47f46fbae95d0439a4305f2ea9dc487e.